csp/tests/impl/test_struct.py

```diff
@@ -171,20 +171,92 @@ def __init__(self, x: int):
 # items[:-2] are normal values of the given type that should be handled,
 # items[-2] is a normal value for non-generic and non-str types and None for generic and str types (the purpose is to test the raise of TypeError if a single object instead of a sequence is passed),
 # items[-1] is a value of a different type that is not convertible to the give type for non-generic types and None for generic types (the purpose is to test the raise of TypeError if an object of the wrong type is passed).
-pystruct_list_test_values = { 
-    int : [4, 2, 3, 5, 6, 7, 8, 's'],
+pystruct_list_test_values = {
```

Comment:
not sure why your linter fixed all this whitespace

Reply:
https://github.com/Point72/csp/pull/211